### PR TITLE
chore(surveys): add basic survey e2e tests

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

Adds some basic survey e2e tests for some form of visual testing

To test:

run cypress tests
## Checklist
- [ ] Tests for new code (see [advice on the tests we use](https://github.com/PostHog/posthog-js#tiers-of-testing))
- [ ] Accounted for the impact of any changes across different browsers

### CODE REVIEW

File: cypress/e2e/surveys.cy.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    questions: [{ type: 'open', question: 'What is your role?' }],
+                },
+            ],
+        }).as('surveys')
+        cy.visit('./playground/cypress')
+        onPageLoad()
+        cy.wait(500)
+        const survey = cy.get('.PostHogSurvey123').shadow()
+        survey.find('.survey-123-form').should('be.visible')
+        survey.find('.question-textarea-wrapper').type('product engineer')
```
Comment (neilkakkar) [inline cypress/e2e/surveys.cy.js:38]: can you also assert the question & description is correct?

And cover all types of surveys?

Comment (neilkakkar) [reply cypress/e2e/surveys.cy.js:38]: and also intercept the `/e/` or `/batch` endpoint to confirm that the right events are sent [dismiss / sent with the right data]

Comment (liyiy) [reply cypress/e2e/surveys.cy.js:41]: The types of surveys should be covered by the multiple question survey test~

Comment (liyiy) [reply cypress/e2e/surveys.cy.js:41]: Should be a good base to build off of for now

Comment (neilkakkar) [reply cypress/e2e/surveys.cy.js:41]: sweet 👍

**Review by neilkakkar — APPROVED** [review]
some event assertions failing but otherwise LGTM!

**Pull request merged**